app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { getAnswerForQuestionAndBuilding } from "../selectors/answersSelector";
+import { getDependentQuestionsForOptions } from "../selectors/questionsSelector";
+
+import DropdownOption from '../components/DropdownOption';
+import RangeOption from '../components/RangeOption';
+import FreeOption from '../components/FreeOption';
+import Question from '../components/Question';
+
+class OptionsContainer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      option_ids: []
+    };
+  }
+
+  handleSelect(option_ids) {
+    this.setState({
```

Comment:
hmmm I'm not sure if using set state is the best way to do this, since this container is directly subscribing from the state tree.. Should the state tree change, then the component should automatically remount